src/Router.php

```diff
@@ -362,11 +362,11 @@ public static function process_http_request() {
 					 * use in the executor.
 					 */
 				} else {
-					$decoded_variables = json_decode( $data['variables'] );
+					$decoded_variables = (array) json_decode( wp_kses_stripslashes( $data['variables'] ) );
```

Comment:
@jasonbahl you can actually pass `true` as the second param of `json_decode()` and it will return an associative array.